### PR TITLE
cleanup(OWNERS): remove inactive approvers

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind cleanup

/kind documentation

**Any specific area of the project related to this PR?**

/area examples

**What this PR does / why we need it**:

For https://github.com/falcosecurity/evolution/issues/157, this PR moves inactive approvers (who had very little or zero contributions over the past 6 months) from `approvers` to `emeritus_approvers` in OWNERS.

Considering the low activity of this repository, devstats have also been consulted for the past year: https://falco.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=falcosecurity%2Fclient-go&var-country_name=All

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by fntlnz — CHANGES_REQUESTED** [review]
Please restore `fntlnz` https://github.com/falcosecurity/falco/issues/2115#issuecomment-1176045086

**Comment by leogr** [comment]
Hey @fntlnz If you decided to step down, could you remove the _change request_ to unblock this PR, please?

@falcosecurity/client-go-maintainers any other thoughts on this?

**Comment by jasondellaluce** [comment]
@fntlnz confirmed his intention to step down as an approver in #58, however neither this PR nor #58 can be merged due to a change request and a missing sign-off respectively, and there has been no response to https://github.com/falcosecurity/client-go/pull/57#issuecomment-1183273388 either.

Closing this in favor of #59 to unblock this changes.

/close

**Pull request closed without merging**
